src/screens/MainApp/DiagnoseRequest/ImageReview/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { View, FlatList } from 'react-native'
+import Carousel, { Pagination } from 'react-native-snap-carousel'
+import Icon from 'react-native-vector-icons/FontAwesome5'
+import { Button } from '~/components'
+import NavigationService from '~/navigationService'
+import ConfirmButton from './ConfirmButton'
+import CarouselImage from './CarouselImage'
+import FlatListImage from './FlatListImage'
+import { requestImages, newImages } from '~/constants/mocks'
+import { MIN_IMAGES, MAX_IMAGES, ADD_MORE_IMAGES_URI } from './constants'
+import styles, { ITEM_HEIGHT, ITEM_WIDTH, CAROUSEL_SLIDER_WIDTH } from './styles'
+
+const ImageVisualization = (props) => {
+  const [activeIndex, setActiveIndex] = useState(0)
+  const [submitError, setSubmitError] = useState(false)
+  const [images, setImages] = useState(requestImages.slice())
+  const carouselRef = useRef()
+  const flatListRef = useRef()
+
+  const setFlatListItem = (index) => {
+    setActiveIndex(index)
+    flatListRef.current.scrollToIndex({ index: index })
+  }
+
+  const setCarouselItem = (index) => {
+    setActiveIndex(index)
+    carouselRef.current.snapToItem(index)
+  }
+
+  const confirmRequest = () => {}
+
+  const getMorePhotos = () => {
+    const filterDuplicates = newImages.filter(element => !images.includes(element))
+    setImages([...images, filterDuplicates[0]])
+  }
+
+  const FlatListItem = ({ item }) => {
+    const itemIndex = images.indexOf(item)
+    const lastIndex = images.length - 1
+    const selected = images[activeIndex] === item
+    if (itemIndex === lastIndex && images.length < MAX_IMAGES) {
```

Comment:
Let's leave a blank line before blocks like this and always before `return` statements (the only exception is when the `return` statement is the only statement within a block).